python/marvin/tools/map.py

```diff
@@ -386,3 +417,257 @@ def plot(self, array='value', xlim=None, ylim=None, zlim=None,
             return (ax, fig)
         else:
             return ax
+
+    def _make_image(self, snr_min, log_cb, use_mask):
+        """Make masked array of image.
+
+        Args:
```

Comment:
And we were able to reach a long-lasting truce about line lengths. Thank god we didn't have a tabs vs. spaces war.